src/cryptonote_core/service_node_list.h

```diff
@@ -100,6 +100,17 @@ namespace service_nodes
     void store(const crypto::public_key &pubkey, cryptonote::Blockchain &blockchain);
   };
 
+  struct pulse_sort_key_t
+  {
+    uint64_t last_height_validating_in_quorum = 0;
+    uint8_t  quorum_index = 0;
+
+    BEGIN_SERIALIZE_OBJECT()
+      VARINT_FIELD(last_height_validating_in_quorum)
+      VARINT_FIELD(quorum_index)
```

Comment:
If you keep this struct (as opposed to a typedef), the quorum_index can just be a regular integer field (no need for varint encoding for something that can't be bigger than a byte).